src/plugins/condition/StyleRuleManager.js

```diff
@@ -182,7 +191,6 @@ export default class StyleRuleManager extends EventEmitter {
 
     destroy() {
         if (this.stopProvidingTelemetry) {
-
             this.stopProvidingTelemetry();
             delete this.stopProvidingTelemetry;
         }
```

Comment:
Just below here is the source of the issue. When `destroy` is called on the `StyleRuleManager`, it unsubscribes from both bounds and editing. This happens if we flip to another component. If `destroy` is run, and the listeners unsubscribe themselves. If we flip back to this component, the constructor isn't called again, so we never resubscribe.

Reply:
Good catch! If this is the issue, then the fix should be in ObjectView.vue. When we navigate to a different object, the ObjectView changes the `currentView`. So we should be tearing down StyleRuleManager when this happens.

I recommend reverting all other changes in this PR and adding the following code here: https://github.com/nasa/openmct/blob/1.7.8/src/ui/components/ObjectView.vue#L120

`        if (this.styleRuleManager) {
            this.styleRuleManager.destroy();
            delete this.styleRuleManager;
        }`

Reply:
A much simpler fix! Thank you @shefalijoshi